tests/aws/services/kms/test_kms.py

```diff
@@ -62,6 +64,37 @@ def _get_alias(kms_client, alias_name, key_id=None):
     return None
 
 
+def enforce_salt_length(signing_algorithm, message_type):
+    kwargs = {}
+
+    if "SHA_256" in signing_algorithm:
+        hasher = hashes.SHA256()
+        salt = 32
+    elif "SHA_384" in signing_algorithm:
+        hasher = hashes.SHA384()
+        salt = 48
+    elif "SHA_512" in signing_algorithm:
+        hasher = hashes.SHA512()
+        salt = 64
+
+    if message_type == "DIGEST":
+        kwargs["algorithm"] = utils.Prehashed(hasher)
+    else:
+        kwargs["algorithm"] = hasher
+
+    if signing_algorithm.startswith("ECDSA"):
+        kwargs["signature_algorithm"] = ec.ECDSA(algorithm=kwargs.pop("algorithm", None))
+        return kwargs
+
+    if signing_algorithm.startswith("RSA"):
+        if "PKCS" in signing_algorithm:
+            kwargs["padding"] = padding.PKCS1v15()
+        elif "PSS" in signing_algorithm:
```

Comment:
we need to add the logic in `kms/models.py`

Reply:
we usually write such logics in `models.py` since this piece of code can be used by other backend parts we shouldn't write it in test files. 

Eg: When we `decrypt` a kms key, we add the logic part of [here](https://github.com/localstack/localstack/blob/e393a7fdeeab4f178540d0be9daf60e438dd68af/localstack/services/kms/models.py#L273-L279) in the `kms/models.py`, and [test](https://github.com/localstack/localstack/blob/e393a7fdeeab4f178540d0be9daf60e438dd68af/tests/aws/services/kms/test_kms.py#L515-L517) just calls the handler which further uses the decrypt logic added in models.

Reply:
This logic is definitely only required in testing. Why would testing-only code go into an application file?

Reply:
This should already be solved here: https://github.com/localstack/localstack/blob/f6c21e30cd112832e4b56cd6831df7e5b947c666/localstack/services/kms/models.py#L394 and https://github.com/localstack/localstack/blob/f6c21e30cd112832e4b56cd6831df7e5b947c666/localstack/services/kms/models.py#L416